components/Common/ShellBox/__tests__/index.test.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { IntlProvider } from 'react-intl';
+import ShellBox from '../index';
+
+const mockWriteText = jest.fn();
+const originalNavigator = { ...window.navigator };
```

Comment:
Nice!

Reply:
Thanks sweetie 😁

Reply:
What?

Reply:
You say nice, I say thanks.

Reply:
Ahm... gotcha. Appreciate that, but I'm not your "sweetie" 😅